utils/upload_nuc_data.py

```diff
@@ -0,0 +1,56 @@
+"""# This is a simple tool to update the cyclus_nuc_data.h5 taken from pyne's upload.py
```

Comment:
Also why is there a `#` at the beginning of this line? I am guessing it used to be a comment.